riscv/insns/vsm4r_vs.h

```diff
@@ -4,7 +4,10 @@
 
 require_vsm4_constraints;
 // No overlap of vd and vs2.
-require(insn.rd() != insn.rs2());
+require_no_overlap_eglmul(insn.rd(), insn.rs2()); \
```

Comment:
The final `\` is not required here:
```suggestion
require_no_overlap_eglmul(insn.rd(), insn.rs2());
```